trustwallet/wallet-core
# Rust sample app

## Description

Sample application in Rust language.

## How to test

- build and run sample app, according to the README.md.
- Check that sample is executed in the 'sample app' CI

<img width="976" alt="image" src="https://user-images.githubusercontent.com/13562139/196388880-17ce555b-502b-493b-bf10-40ff564e8fd9.png">


## Types of changes





* New feature (non-breaking change which adds functionality)



## Checklist



- [ ] Implement proper destructor-cleanup of interface data types (TWData, ...) 

If you're adding a new blockchain
